pkg/server/api/github.go

```diff
@@ -0,0 +1,122 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"github.com/bradleyfalzon/ghinstallation"
+	"github.com/robfig/cron/v3"
+	"github.com/shurcooL/githubv4"
+	"log"
+	"net/http"
+)
+
+const (
+	pageLength = 100
+	cronRate   = "5m"
+)
+
+func (c *Api) getClient() *githubv4.Client {
+	tr := http.DefaultTransport
+
+	itr, err := ghinstallation.NewKeyFromFile(
```

Comment:
this could use a comment why we're doing what we're doing. I assume this is just a set-up so that the githubv4 client lib could use the GitHub App-based authentication?